src/components/UserWarnings/LoggedOut/LoggedOut.tsx

```diff
@@ -1,16 +1,26 @@
-import React from 'react'
+import React, { ReactElement, useEffect, useState } from 'react'
 import { useSelector } from 'react-redux'
 import { BodyLong, Button, Heading, Modal } from '@navikt/ds-react'
 
 import { RootState } from '../../../state/store'
 
-function LoggedOut(): JSX.Element | null {
+function LoggedOut(): ReactElement {
     const loggedOut = useSelector((state: RootState) => state.metadata.loggedOut)
+    const [isModalOpen, setIsModalOpen] = useState(false)
 
-    if (!loggedOut) return null
+    useEffect(() => {
+        if (loggedOut && !isModalOpen) {
+            setIsModalOpen(true)
+        }
+    }, [loggedOut, isModalOpen, setIsModalOpen])
 
     return (
-        <Modal open aria-labelledby="logged-out-header" onClose={() => void 0} closeButton={false}>
+        <Modal
+            open={isModalOpen}
+            aria-labelledby="logged-out-header"
+            onClose={() => setIsModalOpen(false)}
```

Comment:
denne skal ikke være lukkbar, for brukeren har ikke noe grunn til å kunne lukke den. de må enten refreshe for å logge inn på nytt eller lukke tabben :P